packages/react-mutation-mapper/src/store/DefaultMutationMapperStore.ts

```diff
@@ -162,6 +165,11 @@ class DefaultMutationMapperStore implements MutationMapperStore {
         );
     }
 
+    @computed
```

Comment:
i'm not sure this should be computed.  does it reference any observable values inside of it?

Reply:
hmm, right, computed has no effect on this one. I'll just remove it.